game/kernel/jak1/kscheme.cpp

```diff
@@ -1712,12 +1727,10 @@ s32 InitHeapAndSymbol() {
     // check the kernel version!
     auto kernel_version = intern_from_c("*kernel-version*")->value;
     if (!kernel_version || ((kernel_version >> 0x13) != KERNEL_VERSION_MAJOR)) {
-      MsgErr("\n");
-      MsgErr(
-          "dkernel: compiled C kernel version is %d.%d but the goal kernel is %d.%d\n\tfrom the "
-          "goal> prompt (:mch) then mkee your kernel in linux.\n",
-          KERNEL_VERSION_MAJOR, KERNEL_VERSION_MINOR, kernel_version >> 0x13,
-          (kernel_version >> 3) & 0xffff);
+      lg::error("Kernel version mismatch! Compiled C kernel version is {}.{} but the goal kernel "
+                "is {}.{}",
+                KERNEL_VERSION_MAJOR, KERNEL_VERSION_MINOR, kernel_version >> 0x13,
+                (kernel_version >> 3) & 0xffff);
```

Comment:
I made this change while debugging runtime issues. Not using `lg::error` made a problem harder to see.

I don't know if there is some other important reason to use MsgErr instead.